modes/longitudinal/src/toolbarButtons.ts

```diff
@@ -5,32 +5,15 @@ import {
   // ListMenu,
   WindowLevelMenuItem,
 } from '@ohif/ui';
-import { defaults } from '@ohif/core';
+import { defaults, ToolbarService } from '@ohif/core';
+import type { Button, RunCommand } from '@ohif/core/types';
+import { EVENTS } from '@cornerstonejs/core';
```

Comment:
I beleive this means the mode package json should depend on cornerstonejs too

Reply:
Yes, that is true, but it already implicitly does so.  Also, there is no reason one can't declare the values as simple string values.